runs/2020/plzen-jaro-2020/info.yml

```diff
@@ -114,6 +114,7 @@ plan:
   materials:
     - lesson: beginners/exceptions
     - lesson: beginners/modules
+    - lesson: beginners/debugging
```

Comment:
Contents of our course are specified here so I moved `beginners/debugging` from general PyLadies course here.